eldritch/events.py

```diff
@@ -2374,6 +2358,55 @@ def animated(self):
     return True
 
 
+class LoseSpecific(DiscardSpecific):
+  def __init__(self, character, items, to_box=False):
+    super().__init__(character, items, to_box)
+    self._verb = "lose"
+    self._verb_past = "lost"
+
+
+def LoseNamed(character, name, to_box=False):
+  items = values.NamedPossessions(character, name)
+  return LoseSpecific(character, items, to_box)
+
+
+class RollToLose(Event):
+  def __init__(self, character, item):
+    super().__init__()
+    self.character = character
+    self.item = item
+    self.roll = None
+    self.lose = None
+    self.done = False
+
+  def resolve(self, state) -> NoReturn:
+    if self.roll is None:
+      self.roll = DiceRoll(self.character, 1)
```

Comment:
Addendum: if you rebase, you will see some new params on DiceRoll. The args are `name`, which should be set to the name of the item you are rolling for, and `bad`, which should be set to a list of dice rolls that would result in a bad outcome (in this case, `[1]`).

These are UI-only parameters; they should have no effect on the game logic. The `name` param is also available on checks, but it can be omitted if the check/dice roll is the result of an encounter. The `bad` param is needed for any dice roll that is not checking success/failure, though in some cases it may be set to an empty list (i.e. roll a die and lose that much stamina).